src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -5908,22 +5885,6 @@ private void BuildChild(IIndentedStringBuilder writer, XamlMemberDefinition? own
 
 					BuildComplexPropertyValue(writer, owner, null, closureName: outerClosure, generateAssignation: outerClosure != null);
 				}
-				else if (
-					_skipUserControlsInVisualTree
-					&& IsDirectUserControlSubType(knownType)
-					&& HasNoUserControlProperties(xamlObjectDefinition))
-				{
-					writer.AppendLineInvariantIndented("new {0}(skipsInitializeComponents: true).GetContent()", GetGlobalizedTypeName(fullTypeName));
```

Comment:
Searching through the code base, I can't find any constructor that has a parameter named `skipsInitializeComponents`, so if `_skipUserControlsInVisualTree` was true, the generated code will be containing errors. I found that the default is false, and the property `UnoSkipUserControlsInVisualTree` is not documented, so I think this is mostly dead code.

Reply:
It likely is, yes. It probably was removed in Uno 4.